kornia/enhance/jpeg.py

```diff
@@ -344,6 +348,27 @@ def _jpeg_decode(
     return rgb_decoded
 
 
+def _perform_padding(image: Tensor) -> Tuple[Tensor, int, int]:
+    """Pads a given image to be dividable by 16.
+
+    Args:
+        image (Tensor): Image of the shape :math:`(*, 3, H, W)`.
+
+    Returns:
+        image_padded (Tensor): Padded image of the shape :math:`(*, 3, H_{new}, W_{new})`.
+        h_pad (int): Padded pixels along the horizontal axis.
+        w_pad (int): Padded pixels along the vertical axis.
```

Comment:
```suggestion
    Args:
        image: Image of the shape :math:`(*, 3, H, W)`.

    Returns:
        image_padded: Padded image of the shape :math:`(*, 3, H_{new}, W_{new})`.
        h_pad: Padded pixels along the horizontal axis.
        w_pad: Padded pixels along the vertical axis.
```
types can be inferred by the doc